cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -136,8 +137,9 @@ endfunction()
 
 # ----------------------------------------------------------------------
 # ORC Format
+message(STATUS "Using ORC_FORMAT_URL: ${ORC_FORMAT_URL}")
 ExternalProject_Add (orc-format_ep
-  URL "https://www.apache.org/dyn/closer.lua/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz?action=download"
+  URL ${ORC_FORMAT_URL}
   URL_HASH SHA256=739fae5ff94b1f812b413077280361045bf92e510ef04b34a610e23a945d8cd5
```

Comment:
User provided URL may not match the `URL_HASH` below. We should probably avoid passing URL_HASH to ExternalProject_Add